## Import Libraries

In [1]:
import numpy as np
import pandas as pd
import scipy.stats as stats

## Create an array of values

In [2]:
data = np.array([2,5,5,6,12,13,15,17,19,23])

Claculate the z scores for each values in the arary,

In [3]:
stats.zscore(data)

array([-1.46549759, -1.01225091, -1.01225091, -0.86116869,  0.04532467,
        0.19640689,  0.49857134,  0.80073579,  1.10290025,  1.70722915])

Each z scores tells us how many standard deviations away an individual values is from the mean. e.g:
the second value of"5" in the array is -1.012 standarad deviation below the mean. The fift value "12" in the array is 0.045 standarad devition above the mean and the last value "23" in the array is 1.707 datandard deviation above the mean.

Z score normalization refers to the process of normalizing every value in a dataset such as that the mean of all the values is 0 and standarad deviation is 1.